react-motions/src/withConfig.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+function withConfig(Component, config) {
+  const { delay, duration, fillMode } = config;
+
+  if (Component.props.style) {
+    if (delay) {
+      Component.props.style['animationDelay'] = delay
+      Component.props.style['WebKitAnimationDelay'] = delay
```

Comment:
what do you think of the prop `duration` and `delay` being an integer?

and have something like that within the `withConfig`:

```jsx
Component.props.style['animationDelay'] = `${delay}s`
```

Reply:
Btw, the components props are type integer.

Reply:
@emersonlaurentino That would limit setting animation for seconds. `ms` is also a valid value.

Reply:
It's true, we have to review this in the components.

Reply:
@emersonlaurentino Maybe we can just assume that is the default and pass in fractioned numbers to represent milliseconds.

Reply:
i think better pass string with time in ms or s.